src/app.jsx

```diff
@@ -37,14 +37,16 @@ export const Application = () => {
 
     return (
         <NetworkProvider>
-            <Tabs activeKey={activeTabKey} onSelect={handleTabClick}>
-                <Tab eventKey={0} title={<TabTitleText>{_("Interfaces")}</TabTitleText>}>
-                    <InterfacesTab />
-                </Tab>
-                <Tab eventKey={1} title={<TabTitleText>{_("Routing")}</TabTitleText>}>
-                    <RoutingTab />
-                </Tab>
-            </Tabs>
+            <Page id="network-configuration">
+                <Tabs mountOnEnter activeKey={activeTabKey} onSelect={handleTabClick}>
```

Comment:
Now that we are working on removing interfaces/connections, the `mountOnEnter` option might have a nasty side effect. If `useEffect` calls are not processed, the routes will not be fetched, so if you remove an interface its routes will not be changed.

Of course, I think that we should move that `useEffect` to the top-level component (and do pretty much the same for `fetchInterfaces` and `fetchConnections` too).

Reply:
> If `useEffect` calls are not processed, the routes will not be fetched, 

True

> so if you remove an interface its routes will not be changed.

Not sure about it. I mean, if we add logic to delete the routes associated to such interface, they are not going to be fetched when the user revisit the _Routing_ tab because `mountOnEnter`. Maybe [unmountOnExit](https://www.patternfly.org/v4/components/tabs#unmounting-invisible-children) is needed here too.

Anyway, as I'm not completely sure about it, we can just postpone this change for later.

Reply:
You do not need to add the logic there. The point is that the reducer will take care of removing the associated routes when an interface/connection is removed. If the routes are not there yet, well, they will not get removed. We need to routes loaded from the very beginning so we need to move that `fetchRoutes` call up in the tree.

Reply:
On the other hand, you are right that they will not be there if we implement the "deletion" carefully (and we do not trust at all in the state's content).

Reply:
> You do not need to add the logic there. The point is that the reducer will take care of removing the associated routes when an interface/connection is removed.

Yep, that's what I meant. I didn't express it well.

> We need to routes loaded from the very beginning so we need to move that `fetchRoutes` call up in the tree

Now I see the problem. So, let's postpone the use of `mountOnEnter`/`unmountOnExit`.

Reply:
Thanks!